src/testdrive/src/action/s3.rs

```diff
@@ -61,3 +69,45 @@ pub async fn run_verify_data(
 
     Ok(ControlFlow::Continue)
 }
+
+pub async fn run_verify_incomplete(
```

Comment:
Can we make this a bit more general purpose? Feels like this wants to be `s3-verify-missing key=...` or somesuch.

Reply:
How about `s3-verify-keys bucket=<> path-prefix=<> key-pattern=*INCOMPLETE*` ? The goal is to determine whether an object with the specified key is written to the path, not that one is missing

Reply:
Oh, d'oh, I read the code backwards. Yeah, that sounds great!